src/Network-UUID/NilUUID.class.st

```diff
@@ -0,0 +1,13 @@
+"
+UUID with all fields set to 0.
```

Comment:
My point: We don't need a separate class for this :) Just a different instance is enough!

Reply:
NilUUID has been removed :)